features/machine/machine.feature

```diff
@@ -114,3 +114,32 @@ Feature: Machine features testing
     Given I have an IPI deployment
     And evaluation of `BushSlicer::Machine.list(user: admin, project: project('openshift-machine-api'))` is stored in the :machines clipboard
     Then the expression should be true> cb.machines.select{|m|m.instance_state == m.annotation_instance_state}.count == cb.machines.count
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-27609
+  @admin
+  @destructive
+  Scenario: Scaling a machineset with providerSpec.publicIp set to true
+    Given I have an IPI deployment
+    And I clone a machineset named "machineset-clone-publiciptrue"
+    And admin ensures "machineset-clone-publiciptrue" machineset is deleted after scenario
```

Comment:
step `I clone a machineset named "machineset-clone-publiciptrue"` should register clean-up automatically. No need to do that on a separate scenario line.

Reply:
I mean the step needs to be modified to do it.

Reply:
Now this is possible with https://github.com/openshift/verification-tests/pull/780

Reply:
@miyadav Now the delete is auto registered, you don't need to `admin ensures` step, can you please update it?

Reply:
done